ccmlib/common.py

```diff
@@ -52,6 +52,9 @@
 
 DOWNLOAD_IN_PROGRESS_FILE = "download_in_progress"
 
+DEFAULT_DATACENTER = 'datacenter1'
+DEFAULT_RACK = 'rack1'
```

Comment:
this is breaking checks for dtest....

you would need to run all test, and prove there's no dependency